pysma/const.py

```diff
@@ -229,65 +235,400 @@
+    SKEY: "6100_00696E00",
+    SNAME: "battery_capacity_total",
+    SUNIT: "%",
+}
+SENSOR_BATTERY_CAPACITY_A = { # Example: "6100_00499100":{"7":[{"val":100},{"val":null},{"val":null}]}
+    SKEY: "6100_00499100_0",
+    SNAME: "battery_capacity_a",
+    SUNIT: "%",
+}
+SENSOR_BATTERY_CAPACITY_B = { # Example: "6100_00499100":{"7":[{"val":100},{"val":null},{"val":null}]}
+    SKEY: "6100_00499100_1",
+    SNAME: "battery_capacity_b",
+    SUNIT: "%",
+}
+SENSOR_BATTERY_CAPACITY_C = { # Example: "6100_00499100":{"7":[{"val":100},{"val":null},{"val":null}]}
+    SKEY: "6100_00499100_2",
+    SNAME: "battery_capacity_c",
+    SUNIT: "%",
+}
+SENSOR_BATTERY_TEMP_A = { # Example: "6100_40495B00":{"7":[{"val":180},{"val":null},{"val":null}]} # 10904=Bat.TmpVal\4807424\Battery temperature
+    SKEY: "6100_40495B00_0",
+    SNAME: "battery_temp_a",
+    SUNIT: "C",
+    SFACTOR: 10,
+}
+SENSOR_BATTERY_TEMP_B = { # Example: "6100_40495B00":{"7":[{"val":180},{"val":null},{"val":null}]} # 10904=Bat.TmpVal\4807424\Battery temperature
+    SKEY: "6100_40495B00_1",
+    SNAME: "battery_temp_b",
+    SUNIT: "C",
+    SFACTOR: 10,
+}
+SENSOR_BATTERY_TEMP_C = { # Example: "6100_40495B00":{"7":[{"val":180},{"val":null},{"val":null}]} # 10904=Bat.TmpVal\4807424\Battery temperature
+    SKEY: "6100_40495B00_2",
+    SNAME: "battery_temp_c",
+    SUNIT: "C",
+    SFACTOR: 10,
+}
+SENSOR_BATTERY_INSULATION_RESISTANCE = { # Example: # "6102_00254F00":{"7":[{"val":3000000}]} # 10079=Isolation.LeakRis\2445056\Insulation resistance
+    SKEY: "6102_00254F00",
+    SNAME: "battery_insulation_resistance",
+    SUNIT: "kOhm",
+    SFACTOR: 1000,
+}
+SENSOR_BATTERY_INSULATION_RESIDUAL_CURRENT = { # Example: # "6102_40254E00":{"7":[{"val":7}]} # 10078=Isolation.FltA\2444800\Residual current mA
+    SKEY: "6102_40254E00",
+    SNAME: "battery_insulation_residual_current",
+    SUNIT: "mA",
+}
+SENSOR_BATTERY_OPERATING_TIME_TOTAL = { # Example: "6400_00462E00":{"7":[{"val":1314643}]} # 10011=Metering.TotOpTms\4599296\Operating time
+    SKEY: "6400_00462E00",
+    SNAME: "battery_operating_time_total",
+    SUNIT: "h",
+    SFACTOR: 3600,
+}
+SENSOR_BATTERY_OPERATING_TIME_FEEDIN = { # Example: "6400_00462F00":{"7":[{"val":941649}]} # 10012=Metering.TotFeedTms\4599552\Feed-in time
+    SKEY: "6400_00462F00",
+    SNAME: "battery_operating_time_feedin",
+    SUNIT: "h",
+    SFACTOR: 3600,
+}
+SENSOR_BATTERY_STATS_GRID_CONNECTIONS = { # Example: "6402_00666F00":{"7":[{"val":111}]}} # 10010=Operation.GriSwCnt\6713088\Number of grid connections
+    SKEY: "6402_00666F00",
+    SNAME: "battery_stats_grid_connections",
+    SUNIT: "",
+}
+SENSOR_BATTERY_CONDITION_INVERTER = { # Example: "6180_08414C00":{"7":[{"val":[{"tag":307}]}]} # 11176=Operation.Inverter.Health\4279296\Inverter status # Gives back a code, 307 = Ok
+    SKEY: "6180_08414C00",
+    SNAME: "battery_condition_inverter",
+    SPATH: JMESPATHS_TAG,
+    SL10N: True,
+}
```

Comment:
Same here. These are not battery specific, so we should remove the 'battery' from the name.